e2eAIOK/DeNas/asr/model_builder_denas_asr.py

```diff
@@ -1,9 +1,15 @@
 import torch
```

Comment:
From this file, looks like you are embeding prunerin the asr model builder, which should be separated and works independently, since this feature is expect to cover not only ASR domain. 

if you are reusing the previous model builder, you will need a simple API wrapper for the previous code.

Reply:
added `load_pretrained_model_and_prune` function in model builder to load model and call Pruner API